components/js/signal-projects.js

```diff
@@ -1,629 +1,601 @@
-        .append('tr')
-        .selectAll('th')
-        .data(col_array)
-        .enter()
-        .append('th')
-        .text(function(d) {
-            return d;
-        });
-
-    return cols;
-        
+  return cols;
 }
 
-
-
 function createMapSelectors(div_id, obj_arr, display_property, icon_name) {
+  var selectors = d3
+    .select("#" + div_id)
+    .selectAll("div")
+    .data(obj_arr)
+    .enter()
+    .append("div")
+    .attr("class", "col-sm")
+    .append("btn")
+    .attr("type", "button")
+    .attr("data_id", function(d) {
+      return d.name;
+    })
+    .attr("class", "btn btn-block btn-primary btn-map-selector")
+    .attr("aria-pressed", function(d, i) {
+      // class first two buttons as 'active'
+      if (i <= 1) {
+        return true;
+      } else {
+        return false;
+      }
+    })
+    .classed("active", function(d, i) {
+      // class first two buttons as 'active'
+      if (i <= 1) {
+        return true;
+      } else {
+        return false;
+      }
+    })
+    .html(function(d) {
+      return d[display_property];
+    });
 
-    var selectors = d3.select("#" + div_id)
-        .selectAll('div')
-        .data(obj_arr)
-        .enter()
-        .append('div')
-        .attr('class', 'col-sm')
-        .append('btn')
-        .attr('type', 'button')
-        .attr('data_id', function(d){
-            return d.name;
-        })
-        .attr('class', 'btn btn-block btn-primary btn-map-selector')
-        .attr('aria-pressed', function(d, i) {
-            // class first two buttons as 'active'
-            if (i <= 1) {
-                return true;
-            }
-            else {
-                return false;
-            }
-        })
-        .classed('active', function(d, i) {
-            // class first two buttons as 'active'
-            if (i <= 1) {
-                return true;
-            }
-            else {
-                return false;
-            }
-        })
-        .html(function(d){
-            return d[display_property];
-        });
-
-    return selectors;
-    
+  return selectors;
 }
 
-
-
-
 function createMarkers(data, style) {
-    //  create markers and layers for each device type
-
+  //  create markers and layers for each device type
 
-    for (var i = 0; i < data.length; i++) {
-        
-        if (!data[i].location_latitude) {
-            continue;
-        }
-        var status = data[i].signal_status;
-        
-        var marker_style = default_style[status];
+  for (var i = 0; i < data.length; i++) {
+    if (!data[i].location_latitude) {
+      continue;
+    }
+    var status = data[i].signal_status;
 
-        var location_name = data[i].location_name;
+    var marker_style = default_style[status];
 
-        var lat = data[i].location_latitude;
-        
-        var lon = data[i].location_longitude;
+    var location_name = data[i].location_name;
 
-        var updated = formats.formatDate( new Date(data[i].construction_note_date) )
```

Comment:
Sorry if this is hard to read due to all the reformatting, but this section contains the code I actually edited!

Reply:
nice, thanks for highlighting the right section.